src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+@use "../utilis/variables" as *; 
+
+.calendar {
+  display: flex;
+  justify-content: flex-start;
+  width: $width-calendar;
+  flex-wrap: wrap;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    min-width: $width-day;
+    height: $height-day;
+    background-color: $background-color-day;
+    border: 1px solid black;
+
+    margin-top: $margin-day;
+    margin-right: $margin-day;
+
+    cursor: pointer;
+
+    &:hover {
+      background-color: pink;
+      transform: translate(0, -20px);
+      transition: 0.5s;
```

Comment:
Approve.

But pay attention on this point
better to move from hover to &__day selector.
It works in two ways.
And if you not set properties, it will check all properties and affect on performance
`transition: translate 0.5s`